spec/models/quiz_spec.rb

```diff
@@ -5,4 +5,19 @@
 
   it { should validate_presence_of(:name) }
   it { should validate_uniqueness_of(:name) }
+
+  it "returns score for student" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    quiz = Quiz.first
+
+    expect(quiz.student_score(user)).to eq(1)
+  end
+
+  it "returns student's completion date of quiz" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    time = Time.now.round
+    quiz = Quiz.first
```

Comment:
Be more verbose and create a quiz with questions, and then answer submissions for the user.